ultravox/training/evaluation.py

```diff
@@ -44,14 +45,36 @@ def dataset_infer(
     return ddp_utils.all_gather_list(eval_samples)
 
 
-def get_metric_name(ds_name: str, metric: str) -> str:
-    if ds_name == "boolq_in" and metric == "asr":
-        return "boolq__wer"
-    if ds_name == "boolq" and metric == "boolq":
-        return "boolq__correctness"
-    if metric == "instruct":
-        return f"{ds_name}__instruct_follow"
-    return f"{ds_name}__{metric}"
+@dataclasses.dataclass
+class EvalScenario:
+    name: str
+    dataset: str
+    metric: str
+    include_audio: bool = True
+    include_context: bool = True
+    new_tokens: Optional[int] = None
+
+
+EVAL_SCENARIOS = [
+    EvalScenario("boolq__wer", "boolq_in", "asr"),
+    EvalScenario("anyinstruct__instruct_follow", "anyinstruct", "instruct"),
+    EvalScenario(
+        "boolq_binary", "boolq_passage", "exact_match_last_word", new_tokens=128
+    ),
+    EvalScenario(
+        "anyinstruct__instruct_follow_text_only",
+        "anyinstruct",
+        "instruct",
+        include_audio=False,
+    ),
+    EvalScenario(
+        "boolq_binary_text_only",
+        "boolq_passage",
```

Comment:
does this dataset exist?

Reply:
Good catch, I changed that last minute. Seems like I didn't change it everywhere.